sdk/storage/azure_storage_blob/src/models/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+mod blob_properties;
+
+pub use crate::generated::enums::*;
+pub use crate::generated::models::*;
```

Comment:
TODO: Need to start a discussion regarding the way generated code is currently imported. Likely need to bring the scope down to `pub(crate)` and only promote to the root through `lib.rs`.

Currently if this is done (`pub` -> `pub(crate)`), several fields are `pub` but are never accessed, and all warnings are errors and will block CI.